Configuration/Settings.yaml

```diff
@@ -1,5 +1,6 @@
 Flowpack:
   ElasticSearch:
+    version: 2.x
```

Comment:
For us this works well. We run our application on Neow/Flow 4.0.5 with Elasticsearch  5.4.1. We´re looking forward to find this request merged to receive it with the next release - good job, thanks.